backend/carts/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from accounts.models import User
+from products.models import Product
+
+
+class Cart(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
It is a good idea to add related_name="cart" so reverse access is user.cart instead of user.cart_set.
Here is my suggestion:
```suggestion
    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="cart")
```

Reply:
Thank you for the suggestion!
I understand the benefit of using related_name="cart" for reverse access (user.cart instead of user.cart_set).

However, in the current implementation, I'm not accessing the cart from the user side anywhere in the codebase. So adding a related_name would be unnecessary at this point and may add unused complexity.